pyproject.toml

```diff
@@ -45,7 +45,7 @@ license = "MIT"
 name = "hume"
 packages = [{ include = "hume", from = "src" }]
 readme = "README.md"
-version = "0.7.0-rc2"
+version = "0.6.1"
 
 [project.urls]
 Documentation = 'hhttps://dev.hume.ai/reference'
```

Comment:
```suggestion
Documentation = 'https://dev.hume.ai'
```

Reply:
We should handle this in the config repo
https://github.com/HumeAI/fern-config/pull/407/files